.gitmodules

```diff
@@ -1,3 +1,4 @@
 [submodule "vendor/cpp-driver"]
 	path = vendor/cpp-driver
-	url = git@github.com:datastax/cpp-driver.git
+	url = https://github.com/scylladb/cpp-driver.git
+	branch = master
```

Comment:
I think we can use version https://github.com/scylladb/cpp-driver/tree/2.15.2-1
I set master for test because acsylla under development at this time